main.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield self[i]
+      i += 1
+    end
+  end
+
+  def my_each_with_index
+    i = 0
+    while i < length
+      yield i, self[i]
+      i += 1
+    end
+  end
+
+  def my_select
+    array = []
+    i = 0
+    while i < length
+      array << self[i] if yield(self[i])
+      i += 1
+    end
+    array
+  end
+
+  def my_all?
+    all_true = true
+    i = 0
+    while i < length
+      all_true = false unless yield(self[i])
+      i += 1
+    end
+    all_true
+  end
+
+  def my_any?
+    any_true = false
+    i = 0
+    while i < length
+      any_true = true if yield(self[i])
+      i += 1
+    end
+    any_true
+  end
+
+  def my_none?
+    all_false = true
+    i = 0
+    while i < length
+      all_false = false if yield(self[i])
+      i += 1
+    end
+    all_false
+  end
+
+  def my_count
```

Comment:
Just one changes, and get a thumps up...
I got wrong output when i ran your code, error message below; 
![image](https://user-images.githubusercontent.com/31897434/63983050-ed92e400-cabc-11e9-8b05-bffbc11999ad.png)